kustomize/monitoring/base/monitoring.yaml

```diff
@@ -125,6 +125,93 @@ data:
       - source_labels: [__meta_kubernetes_service_name]
         action: replace
         target_label: kubernetes_name
+    - job_name: kfserving-monitoring
+      honor_timestamps: true
+      scrape_interval: 15s
+      scrape_timeout: 10s
+      metrics_path: /metrics
+      scheme: http
+      follow_redirects: true
+      relabel_configs:
+      - source_labels: [__meta_kubernetes_service_label_networking_internal_knative_dev_serviceType]
+        separator: ;
+        regex: Private
+        replacement: $1
+        action: keep
+      - source_labels: [__meta_kubernetes_endpoint_port_name]
+        separator: ;
+        regex: http-usermetric
+        replacement: $1
+        action: keep
+      - source_labels: [__meta_kubernetes_endpoint_address_target_kind, __meta_kubernetes_endpoint_address_target_name]
+        separator: ;
+        regex: Node;(.*)
+        target_label: node
+        replacement: ${1}
```

Comment:
What is $1 & ${1} ?

Don't we create our own labels?

Reply:
I have added deployment name and the component name. We can add more if needed